.github/renovate.json5

```diff
@@ -3,23 +3,28 @@
   extends: [
     'config:recommended',
     'docker:pinDigests',
-    'helpers:pinGitHubActionDigests',
+    'helpers:pinGitHubActionDigestsToSemver',
   ],
   ignorePresets: [
     ':ignoreModulesAndTests', // needed to keep maven-extension test pom files up-to-date
   ],
-  prHourlyLimit: 5,
+  prHourlyLimit: 5, // we have a large number of parallel runners
+  labels: [
+    'dependencies'
+  ],
   packageRules: [
     {
-      // this is to reduce the number of renovate PRs
-      matchManagers: [
-        'github-actions',
-        'dockerfile',
-      ],
-      extends: [
-        'schedule:weekly',
-      ],
-      groupName: 'weekly update',
+      // reduces the number of Renovate PRs
+      // (patch updates are typically non-breaking)
+      "groupName": "all patch versions",
+      "matchUpdateTypes": ["patch"],
+      "schedule": ["before 8am every weekday"]
+    },
+    {
+      // avoids these Renovate PRs from trickling in throughout the week
+      // (consolidating the review process)
+      "matchUpdateTypes": ["minor", "major"],
+      "schedule": ["before 8am on Monday"]
```

Comment:
Hmmmm, I kinda like having them trickle in because I assume that it helps spread the work across approver availability more evenly...but I dunno. This still separates packges into separate PRs tho right, not one big PR monday monring?

Reply:
yeah, that makes sense, but this also rings true

> Receiving PRs at any hour can increase the feeling of being "overwhelmed" by updates and possibly interrupt your flow during working hours, so many Renovate users also consider reducing Renovate's schedule to be outside their normal working hours, for example weeknights and weekends. 

https://docs.renovatebot.com/noise-reduction/

I kind of like the idea of dealing with renovate PRs in the morning and not having them trickle in, but not sure how to balance that across multiple maintainers and multiple definitions of morning